src/repository/screens/repository-file.screen.js

```diff
@@ -11,6 +11,10 @@ import { Card, Icon } from 'react-native-elements';
 
 import { ViewContainer, LoadingContainer } from 'components';
 
+import SyntaxHighlighter from 'react-native-syntax-highlighter';
+import { getLanguage } from 'lowlight';
+import { github as GithubStyle } from 'react-syntax-highlighter/dist/styles';
+
```

Comment:
I put these blanks lines because there are some others in the "_imports part_" and I didn't find the logic behind that... So... I added some spaces 😛.

I'll remove them.